github.sgnl.yaml

```diff
@@ -640,6 +654,579 @@ entities:
+  IssueAssignee:
+    displayName: IssueAssignee
+    externalId: IssueAssignee
+    description: Entity representing the connection between a User entity assigned to a GitHub Issue
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: uniqueId
+        externalId: uniqueId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: issueId
+        externalId: issueId
+        type: String
+        indexed: true
+        attributeAlias: assignedIssueId
+      - name: userId
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: issueAssigneeId
+      - name: login # remove later
+        externalId: login
+        type: String
+  IssueParticipant:
+    displayName: IssueParticipant
+    externalId: IssueParticipant
+    description: Entity representing the connection between a User entity that is a participant of a GitHub Issue
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: uniqueId
+        externalId: uniqueId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: issueId
+        externalId: issueId
+        type: String
+        indexed: true
+        attributeAlias: participantIssueId
+      - name: userId
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: issueParticipantId
+      - name: login # remove later
+        externalId: login
+        type: String
+  
+  # https://docs.github.com/en/graphql/reference/objects#label
+  Label:
+    displayName: Label
+    externalId: Label
+    description: Entity representing a GitHub Label
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: labelId
+      - name: color
+        externalId: color
+        type: String
+      - name: createdAt
+        externalId: createdAt
+        type: DateTime
+      - name: isDefault
+        externalId: isDefault
+        type: Bool
+      - name: name
+        externalId: name
+        type: String
+      - name: updatedAt
+        externalId: updatedAt
+        type: DateTime
+  RepositoryLabel: # TODO: Might want to remove this from catalog completely
+    displayName: RepositoryLabel
+    externalId: RepositoryLabel
+    description: Entity representing the metadata of a Label entity part of a GitHub Repository
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: uniqueId
+        externalId: uniqueId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: repositoryId
+        externalId: repositoryId
+        type: String
+        indexed: true
+        attributeAlias: repositoryLabelRepositoryId
+      - name: labelId
+        externalId: id
+        type: String
+        indexed: true
+        attributeAlias: repositoryLabelId
+      - name: name # remove later
```

Comment:
Is this required?
I see a "remove later" comment

Reply:
Not required, I'm going to remove these 'name/login' attributes on the connection entities after testing in neo4j, it's just added in there for my convenience. I'll remove them before merging

Reply:
Please maintain a copy of testing catalog in a different branch for personal/testing use. It is good practice to exclude testing code in pull requests (unless the test is a part of the PR).